src/store/GlobalContext.jsx

```diff
@@ -29,12 +29,7 @@ function GlobalProvider({ children }) {
   const handleGeolocate = async () => {
     try {
       const location = await useCurrentLocation();
```

Comment:
useCurrentLocation isn't a hook by implementation. Please rename and move from the hooks folder